### PR TITLE
Group people based on the chosen metric

### PR DESCRIPTION
**Pull request by elayabharath**
+ Fixed the idp_status key

### CODE REVIEW

File: lib/model.dart [hunk 1]
```diff
@@ -10,19 +10,19 @@ class Person {
   num age;
   String age_category;
   String gender;
-  String IDPStatus;
+  String idp_status;
```
Comment (lukechurch) [inline lib/model.dart]: idpStatus would be conventional in Dart?

File: lib/interactive.dart [hunk 2]
```diff
@@ -155,6 +155,8 @@ class Interactive {
       ..setAttribute("y2", "${CHART_HEIGHT - CHART_XAXIS_HEIGHT}");
     svgContainer.append(xAxisLine);
 
+    var groupPeople = Map<String, List<model.Person>>();
```
Comment (lukechurch) [inline lib/interactive.dart]: groupPeople doesn't really make sense to me.

peopleByLabel?

**Review by lukechurch — APPROVED** [review]
Thanks @elayabharath!

File: lib/model.dart (hunk 1 above)
Comment (elayabharath) [reply lib/model.dart]: Fixed

File: lib/interactive.dart (hunk 2 above)
Comment (elayabharath) [reply lib/interactive.dart]: Yes peopleByLabel is much better - will take note on the variable names. Fixed

**Pull request merged**